script.js

```diff
@@ -0,0 +1,34 @@
+import request from 'superagent'
+
+function onSubmit (event) {
+  event.preventDefault()
+  console.log('submitted!')
+  const query = document.getElementById('query').value
+  getData(query)
+}
+
+function getData (username) {
+  request.get(`https://api.github.com/users/${username}`)
+    .then(function (result) {
+      const url = (result.body.url)
```

Comment:
Having the right-hand side in parentheses doesn't hurt, but it doesn't do anything, either. I recommend leaving them off in the future.